test/Unread-test.ts

```diff
@@ -0,0 +1,113 @@
+/*
+Copyright 2022 The Matrix.org Foundation C.I.C.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import { mocked } from "jest-mock";
+import {
+    MatrixEvent,
+    EventType,
+    MsgType,
+} from "matrix-js-sdk/src/matrix";
+
+import { haveRendererForEvent } from "../src/events/EventTileFactory";
+import { getMockClientWithEventEmitter, mockClientMethodsUser } from "./test-utils";
+import { eventTriggersUnreadCount } from "../src/Unread";
+
+jest.mock("../src/events/EventTileFactory", () => ({
+    haveRendererForEvent: jest.fn(),
+}));
+
+describe('eventTriggersUnreadCount()', () => {
+    const aliceId = '@alice:server.org';
+    const bobId = '@bob:server.org';
+
+    // mock user credentials
+    getMockClientWithEventEmitter({
+        ...mockClientMethodsUser(bobId),
+    });
+
+    // setup events
+    const alicesMessage = new MatrixEvent({
+        type: EventType.RoomMessage,
+        sender: aliceId,
+        content: {
+            msgtype: MsgType.Text,
+            body: 'Hello from Bob',
```

Comment:
I think this should be

```suggestion
            body: 'Hello from Alice',
```